samples/CommunityToolkit.Maui.Sample/App.xaml

```diff
@@ -65,9 +66,9 @@
             <Style x:Key="label_section_header" TargetType="Label">
                 <Setter Property="TextColor" Value="{StaticResource HeadingLabelTextColor}" />
             </Style>
-            
+
             <!--  Values  -->
-            <Thickness x:Key="ContentPadding">20, 0</Thickness>
+            <maui:Thickness x:Key="ContentPadding">20, 0</maui:Thickness>
```

Comment:
This should be in the default `xmlns`, no? Is it a bug?

Reply:
Honestly - I'm not a XAML expert and have no clue.

When I opened the file in VS for Mac, it had a red squiggle under `<Thickness x:Key="ContentPadding">20, 0</Thickness>`. 

When I clicked the lightbulb in the IDE, it recommended this....so I did it 😊

Reply:
I'll try removing it and see if the Azure DevOps pipelines still work 👍

Reply:
Done ✅ 

Looks like it was just a false negative in VS for Mac